python/paddle/fluid/executor.py

```diff
@@ -306,7 +306,7 @@ def run(self,
             program = default_main_program()
 
         if not isinstance(program, Program):
-            raise TypeError()
+            raise TypeError("executor take Program as arguments.")
```

Comment:
I don't quite understand this message -- should an executor take a Program instance as its argument or not?  If the answer is yes, this message should be

> An executor requires Program as its parameter, but we passed in ...

If the answer is no, the message should be

> An executor doesn't accept Program as its parameter. Please pass in ...

Reply:
Thanks for pointing this out. fixed.